### PR TITLE
Version bump Opendistro 0.10.0.4

### PR DESCRIPTION
**Pull request by sujithvm**

### CODE REVIEW

File: opendistro-elasticsearch-security.release-notes [hunk 1]
```diff
@@ -1,4 +1,8 @@
-## 2019-09-11, Version 0.10.0.3 (Current)
+## 2020-01-29, Version 0.10.0.4 (Current)
+
+- New API to fetch and update account details (currently used for password update)
+
+## 2019-09-11, Version 0.10.0.3
```
Comment (debjanibnrj) [inline opendistro-elasticsearch-security.release-notes:5]: remove this

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    runs-on: ubuntu-latest
+    container: opendistroforelasticsearch/security-maven:v1
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install
+        run: mvn clean install
```
Comment (debjanibnrj) [inline .github/workflows/ci.yml]: minor add new line

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - master
```
Comment (debjanibnrj) [inline .github/workflows/ci.yml:9]: shouldn't this be opendistro-0.10?

File: opendistro-elasticsearch-security.release-notes (hunk 1 above)
Comment (sujithvm) [reply opendistro-elasticsearch-security.release-notes:5]: Why remove this? Its part of the history

File: .github/workflows/ci.yml (hunk 3 above)
Comment (sujithvm) [reply .github/workflows/ci.yml:9]: These are triggers. Checkout action will checkout the current PR only

File: opendistro-elasticsearch-security.release-notes (hunk 1 above)
Comment (debjanibnrj) [reply opendistro-elasticsearch-security.release-notes:5]: got it, ignore the previous comment.

**Review by debjanibnrj — APPROVED** [review]

**Pull request merged**